tests/test_openjdk_devel.py

```diff
@@ -93,3 +116,27 @@ def test_entrypoint(container, java_version, host, container_runtime):
 
     assert "jshell>" in intro
     assert f"Welcome to JShell -- Version {java_version}" in intro
+
+
+@pytest.mark.parametrize(
+    "container",
+    CONTAINER_IMAGES_EXTENDED,
+    indirect=["container"],
+)
+@pytest.mark.parametrize(
+    "java_file",
+    ["Basic"],
+)
+def test_compile(container, java_file, host, container_runtime):
+    """Verify that the entry point of the OpenJDK devel container is the
+    :command:`jshell`.
+
+    """
+    container.connection.run_expect(
+        [0],
+        f"ls {container_test_dir}{java_file}.java",
+    )
+    container.connection.run_expect(
+        [0],
+        f"javac {container_test_dir}{java_file}.java",
```

Comment:
Silly question, does this only compile the code? Wouldn't it make sense to also run it?

Reply:
the idea is to try the compiler... we could try to run the compiled file

Reply:
done